app/src/main/java/com/kabouzeid/gramophone/model/lyrics/SynchronizedLyricsLRC.java

```diff
@@ -0,0 +1,72 @@
+package com.kabouzeid.gramophone.model.lyrics;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+class SynchronizedLyricsLRC extends AbsSynchronizedLyrics {
+    private static final Pattern LRC_LINE_PATTERN = Pattern.compile("((?:\\[.*?\\])+)(.*)");
+    private static final Pattern LRC_TIME_PATTERN = Pattern.compile("\\[(\\d+):(\\d{2}(?:\\.\\d+)?)\\]");
+    private static final Pattern LRC_ATTRIBUTE_PATTERN = Pattern.compile("\\[(\\D+):(.+)\\]");
+
+    private static final float LRC_SECONDS_TO_MS_MULTIPLIER = 1000f;
+    private static final int LRC_MINUTES_TO_MS_MULTIPLIER = 60000;
+
+    @Override
+    public SynchronizedLyricsLRC parse(boolean check) {
+        if (this.parsed || this.data == null || this.data.isEmpty()) {
+            return this;
+        }
+
+        String[] lines = this.data.split("\r?\n");
+
+        for (String line : lines) {
+            line = line.trim();
+            if (line.isEmpty()) {
+                continue;
+            }
+
+            Matcher attrMatcher = SynchronizedLyricsLRC.LRC_ATTRIBUTE_PATTERN.matcher(line);
+            if (attrMatcher.find()) {
+                try {
+                    String attr = attrMatcher.group(1).toLowerCase().trim();
+                    String value = attrMatcher.group(2).toLowerCase().trim();
+                    switch (attr) {
+                        case "offset":
```

Comment:
There are some other attributes like artist, album, title, duration, etc. but I think only offset is useful. Should I leave it as is in case some another attribute needs to be handled, replace `switch` with `if` or replace `LRC_ATTRIBUTE_PATTERN` so it handles only `\[offset: -?\d+\]`?